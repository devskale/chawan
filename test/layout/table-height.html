<table height=1>
<tr><td>
test<br>
test<br>
test
</td></tr>
</table>
after table
<table style="height: 5em">
<tr><td>
test<br>
test<br>
test
</td></tr>
</table>
after table 2
