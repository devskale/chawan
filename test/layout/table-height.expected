 test
 test
 test
after table
 test
 test
 test


after table 2
